shared/constants/network.ts

```diff
@@ -243,34 +246,31 @@ export const INFURA_PROVIDER_TYPES = [
   NETWORK_TYPES.MAINNET,
   NETWORK_TYPES.GOERLI,
   NETWORK_TYPES.SEPOLIA,
+  NETWORK_TYPES.LINEA_GOERLI,
 ] as const;
 
 export const TEST_CHAINS = [
   CHAIN_IDS.GOERLI,
   CHAIN_IDS.SEPOLIA,
-  CHAIN_IDS.LINEA_TESTNET,
+  CHAIN_IDS.LINEA_GOERLI,
   CHAIN_IDS.LOCALHOST,
 ];
 
 const typedCapitalize = <K extends string>(k: K): Capitalize<K> =>
   capitalize(k) as Capitalize<typeof k>;
 
 export const TEST_NETWORK_TICKER_MAP: {
-  [K in Exclude<
-    NetworkType,
-    'localhost' | 'mainnet' | 'rpc'
-  >]: `${Capitalize<K>}${typeof CURRENCY_SYMBOLS.ETH}`;
+  [K in Exclude<NetworkType, 'localhost' | 'mainnet' | 'rpc'>]:
+    | `${Capitalize<K>}${typeof CURRENCY_SYMBOLS.ETH}`
+    | string;
```

Comment:
Nit: this type union can be simplified (the template is a subset of `string`, so this effectively collapses to `string`)
```suggestion
  [K in Exclude<NetworkType, 'localhost' | 'mainnet' | 'rpc'>]: string;
```